src/third-party-tags/inizio.ts

```diff
@@ -5,9 +5,10 @@ const handleQuerySurveyDone = (
 	survey: { measurementId: string },
 ): void => {
 	if (surveyAvailable) {
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- @types/googletag declares it, but it may be missing
```

Comment:
This is slightly annoying, but I think it’s worth keeping our types in sync.

Reply:
I'm curious about this, is it only possibly undefined in this particular case? Because throughout the rest of the source in frontend we removed the `window.googletag?.` optional chaining?

Reply:
In this case, we aren’t mocking it, whereas in `frontend` it’s always mocked (in tests) or actually imported (in prod).

Here the tests are failing if it’s not defined.

Reply:
Ah gotcha, that makes sense!